### PR TITLE
Added more values to the defaults

### PR DESCRIPTION
**Pull request by CameronTEllis**
Defaults added both within the argparser as well as outside of it. This will make the input of the data_dict into generate_data optional

### CODE REVIEW

**Comment by CameronTEllis** [comment]
@mihaic this is crashing because of other code I didn't change. Did something change in pytest to make it not like `l` as a variable name?

**Comment by mihaic** [comment]
It's probably the new release of Flake8. Looking into it.

**Comment by mihaic** [comment]
@CameronTEllis, the errors should go away if you get the latest commit from master.

**Comment by CameronTEllis** [comment]
@gdoubleyew Can you review this please to see if it is suitable?

**Comment by mihaic** [comment]
@gdoubleyew, in case you are wondering about the coverage failure, I think it is a fluke, so please ignore it.

File: brainiak/utils/fmrisim_real_time_generator.py [hunk 1]
```diff
@@ -244,7 +244,8 @@ def _write_dicom(output_name,
 
 def _get_input_names(data_dict):
```
Comment (gdoubleyew) [inline brainiak/utils/fmrisim_real_time_generator.py:245]: maybe we could rename this function to _get_input_defaults() ?

**Pull request merged**